utils/calc_total_unclaimed_fees.go

```
@@ -0,0 +1,54 @@
+package utils
+
+import (
+	"log"
+	"math/big"
+	"strconv"
+)
+
+var tokenSymbolToName = map[string]string{
+	"WETH": "ethereum",
+	"DAI":  "dai",
+	"UNI":  "uniswap",
+}
+
+// calculates total unclaimed fees for the pool
+func CalcTotalUnclaimedFees(
+	token0_unclaimed_fees string,
+	token1_unclaimed_fees string,
+	token0_name string,
+	token1_name string,
+) string {
```

This should return an error instead of printing